libdevcore/Common.h

```diff
@@ -37,6 +37,10 @@
 #pragma warning(disable:3682) //call through incomplete class
 #endif
 
+#ifdef __MIC__
```

Comment:
This is actually part of c++11: http://en.cppreference.com/w/cpp/language/operator_alternative